doc/client_lua_api.txt

```diff
@@ -30,6 +30,16 @@ Startup
 Mods are loaded during client startup from the mod load paths by running
 the `init.lua` scripts in a shared environment.
 
+In order to load client-side mods in your worlds, you have 2 conditions :
```

Comment:
IMO, it's bad practice to use pronouns like "you" in documentation. Using passive voice might be a better solution here.

```suggestion
In order to load client-side mods in a world, the following conditions need to be satisfied:
```